frontend/config-assistant/src/components/toolbar/fetchWebSchemas.ts

```diff
@@ -1,12 +1,12 @@
 import type {SchemaOption} from '@/model/SchemaOption';
 import {JSON_SCHEMA_STORE_CATALOG_URL} from '@/constants';
 
-export async function fetchWebSchemas(): Promise<SchemaOption[]> {
+export async function fetchWebSchemas(toast?: any): Promise<SchemaOption[]> {
   let fetchedSchemas: SchemaOption[] = [];
-
   const response = await fetch(JSON_SCHEMA_STORE_CATALOG_URL);
   const data = await response.json();
   const schemas = data.schemas;
+  const schemaName = schemas.label || 'Unknown Schema';
```

Comment:
Does this make sense? When you have a list of schemas, you don't have one particular schema name, right?

Reply:
this schema name is when toast arrives, toast name is necessary in this case. We have list of schemas and we click on one schema and for that we need name so that we can toast that

Reply:
its done, and I think its alright when we merge this

Reply:
have you posted a video of this? I cannot try it out right now

Reply:
here I Post Video for that, please observe **toast** after clicking schema. its too big, I will post in our group.

Reply:
@Keyshal-211 I still think this does not make sense. The function fetches a list of schemas from JSON Schema store, but it does not select or fetch any individual schema. What value would schemaName have? It comes from schemas.label und schemas is the collection of schemas.